spring-cloud-deployer-local/src/test/java/org/springframework/cloud/deployer/spi/local/JavaExecutionCommandBuilderTests.java

```diff
@@ -27,15 +28,18 @@
 
 import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasItem;
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.fail;
```

Comment:
Same her on the static imports.  At a minimum, they shouldn't be nested in the middle of the imports.

Reply:
Probably my IDE is not configured, I always run `mvn install -Pfull` which usually what triggers the checkstyle plugin, I was not aware it was not configured here